src/main/scala/tech/cryptonomic/conseil/tezos/michelson/JsonToMichelson.scala

```diff
@@ -12,4 +15,26 @@ object JsonToMichelson {
 
   def convert[T <: MichelsonElement: Parser](json: String): Result[String] =
```

Comment:
I think I only moved one method in this class because I needed to re-use it.

Probably it was already quite similar to the other, I'll look into that and see if removing it breaks anything